src/ng-currency.directive.js

```diff
@@ -146,13 +154,21 @@ export default function ngCurrency($filter, $locale, $timeout, ngCurrencySetting
 
       element.bind('focus', () => {
         if (active) {
+          if (autoFill && [undefined, null, ''].indexOf(controller.$$rawModelValue) !== -1) {
+            controller.$$rawModelValue = '0';
```

Comment:
This makes it so once the user focuses the field the value is populated with zero (if its empty and autoFill is enabled).

Instead we should prepopulate it with zero.